ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -117,6 +117,7 @@ descriptor:
   panel: "An interface to control a device."
   fabric: "Textile material."
   failed: "Indicates that a point is not operating as intended or is incapable of operating."
+  filtration: "Component used for removing dust and other particulate matter from the air."
```

Comment:
This is not a good description, we need to differentiate this from `filter`. Please update.